libminifi/src/c2/C2Agent.cpp

```diff
@@ -539,9 +563,9 @@ void C2Agent::handle_describe(const C2ContentResponse &resp) {
   } else if (resp.name == "manifest") {
     auto keys = configuration_->getConfiguredKeys();
     C2Payload response(Operation::ACKNOWLEDGE, resp.ident, false, true);
-    response.setLabel("configuration_options");
+    response.setLabel("configurationOptions");
```

Comment:
Why does this have the same label as the response for "configuration"?
It would also have the same label as the response for "jstack", but "jstack" hasn't been changed to "configurationOptions".
This does not look right.

Reply:
For jstack "configuration_options" does not appear in the response payload. The inner payload label matters while serializing json. That is what I found out while testing DESCRIBE jstack. 
On the side note, the DESCRIBE operand should be pstack I believe for a better name for C++. Looks like jstack was taken from Minifi Java.
Nonetheless, removed that label for jstack response.